wadsrc/static/zscript/actors/heretic/hereticarmor.zs

```diff
@@ -7,6 +7,7 @@ Class SilverShield : BasicArmorPickup
 	{
 		+FLOATBOB
 		Inventory.Pickupmessage "$TXT_ITEMSHIELD1";
+		Inventory.Pickupmessage "$TAG_ITEMSHIELD1";
```

Comment:
Should be `Tag` instead of `Inventory.Pickupmessage`.

Reply:
Woops, copy-paste mishap there. Thanks!